pkg/cmd/job/job_add.go

```diff
@@ -0,0 +1,72 @@
+package job
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-core-cli/pkg/metronome"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+)
+
+// newCmdClusterAdd creates a new job.
+func newCmdJobAdd(ctx api.Context) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
```

Comment:
When there is an arg we're adding it here, see https://github.com/dcos/dcos-cli/blob/96375b35f1d379f092f23a5e4f677a0b73200c1e/pkg/cmd/auth/auth_listproviders.go#L17